openfisca_uk/parameters/benefits/LHA/means_test/worker_hours.yaml

```diff
@@ -0,0 +1,6 @@
+description: Default hours requirement
```

Comment:
For consistency please add a period here (and other descriptions), or remove from others.